xml/dashboard_user_cmdline.xml

```diff
@@ -100,8 +100,8 @@
   </para>
 
   <para>
-   User accounts are stored in &mon;’s configuration database and are
-   globally shared across all &mgr; instances.
+   User accounts are stored in &mon;’s configuration database and are globally
+   shared across all &mgr; instances.
```

Comment:
```suggestion
   User accounts are stored in &mon;'s configuration database and are shared
   globally across all &mgr; instances.
```

Reply:
Be very careful about accidentally inserting curly ‘single’ and “double” quotation marks. Our rendering and production software inserts these automatically, but it expects 'straight' quotes, like those, and not curly ones, \`backticks\` or isolated acute accents: ´

Reply:
I made no changes to that part of the documentation with my original commit here? Or what am I missing?

Reply:
I had to check, but you are right. Sorry for the noise. 

For clarity: are we talking about one single MON node or several MON nodes?